travis-ymls/rxJava.travis.yml

```diff
@@ -0,0 +1,44 @@
+  
+# ----------------------------------------------------------------------------
+#
+# Package             : rxJava
+# Source Repo         : https://github.com/ReactiveX/RxJava
+# Travis Job Link     : https://travis-ci.com/github/genisysram/RxJava/jobs/488191823
+# Created travis.yml  : Yes
+# Maintainer          : <ramakrishna.s@genisys-group.com>
+#                       <sethp@us.ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+
+
+language: java
+jdk:
+- openjdk8
+- openjdk11
+
+arch:
+    - amd64
+    - ppc64le
+
+install: true
+
+sudo: required
+
+# script for build and release via Travis to Bintray
+script: 
+    - ./gradlew
```

Comment:
You are still using just this command. I think it's not doing anything as per logs.